VERSION.txt

```diff
@@ -1 +1 @@
-2.1-dev
+2.1.0-dev
```

Comment:
I wasn't aware of that, sorry. I assumed we were using [semver](http://semver.org/). I'll change it.